### PR TITLE
DOC-4395

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-4395

### CODE REVIEW

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 1]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
...
+|TODO
+
+|1.5
+|TODO
+
+|2.0
+|TODO
+
+|2.1
+|TODO
```
Comment (jamesnocentini) [inline modules/ROOT/pages/sgcollect-info.adoc]: @adamcfraser @bbrks ^^^ this table needs info.

Comment (jamesnocentini) [reply modules/ROOT/pages/sgcollect-info.adoc]: this table is rendered here https://docs.couchbase.com/sync-gateway/2.1/sgcollect-info.html#installing-optional-dependencies (currently missing Go versions for SG 1.4 and above)

Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.8.5
```

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 2]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
...
 |1.6.3
+
+|1.4
+|TODO
+
+|1.5
+|TODO
+
+|2.0
+|TODO
```
Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.8.5
```

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 3]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
 
 |1.3.1
 |1.6.3
+
+|1.4
+|TODO
+
+|1.5
+|TODO
```
Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.7.4
```

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 4]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
 
 |1.3.1
 |1.6.3
+
+|1.4
+|TODO
+
+|1.5
```
Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.5.2
```

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 5]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
 
 |1.3.1
 |1.6.3
+
+|1.4
+|TODO
```
Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.7.4
```

File: modules/ROOT/pages/sgcollect-info.adoc [hunk 6]
```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
 
 |1.3.1
 |1.6.3
+
+|1.4
```
Comment (adamcfraser) [inline modules/ROOT/pages/sgcollect-info.adoc]:
```suggestion
|1.4.1
```

**Comment by adamcfraser** [comment]
Made suggestions for go versions.  Switched to SG versions 1.4.1 and 1.5.2, let me know if that's a problem.

**Comment by jamesnocentini** [comment]
That's great! Thanks

**Pull request merged**